openghg/retrieve/_access.py

```diff
@@ -23,29 +21,33 @@
 
 DataTypes = Union[BoundaryConditionsData, FluxData, FootprintData, ObsColumnData, ObsData]
 multDataTypes = Union[
-    List[BoundaryConditionsData], List[FluxData], List[FootprintData], List[ObsColumnData], List[ObsData]
+    list[BoundaryConditionsData], list[FluxData], list[FootprintData], list[ObsColumnData], list[ObsData]
 ]
 
 
 def _get_generic(
-    sort: bool = True,
+    combine_multiple_inlets: bool = False,
     elevate_inlets: bool = False,
```

Comment:
I think we can probably just remove `elevate_inlets` entirely and replace with `combine_multiple_inlets` if that's easier. I wouldn't say anyone using the code or other code is relying on it at the moment.

Reply:
Okay, I'll remove it.